2023/index.md

```diff
@@ -99,10 +99,113 @@ The 3rd NLP-OSS workshop will be co-located with the EMNLP 2023 conference.
 <br><br><br><br>
 
 
-<!-- ## Workshop Program
+## Workshop Program
 <br><br>
 
-The timezone for the program schedule below are in **Singapore Time (GMT +8)**.  -->
+The timezone for the program schedule below are in **Singapore Time (GMT +8)**.
+
+<p>09:00 - 09:15  &nbsp;&nbsp; <strong> Opening Remarks </strong> </p>
+<p>09:15 - 10:15 &nbsp;&nbsp; <strong> Invited Talk  - SEA-LION (Southeast Asian Languages In One Network): A Family of Southeast Asian Language Models </strong> <br>
+<em>William Tjhi, David Ong and Peerat Limkonchotiwat</em></p>
```

Comment:
For this invited talk, we've confirmed only David and Peerat. William and other SEA-LION collaborators are busy so they will not be able to present. So I think for now, we'll have David and Peerat only.

But William and others from SEA-LION might attend the workshop, we should gladly introduce them if they do =)